src/generator-attribute-manager.cs

```diff
@@ -198,7 +198,7 @@ static IEnumerable<System.Attribute> ConvertOldAttributes (CustomAttributeData a
 					arr [a] = (Type) typed_values [a].Value;
 				value = arr;
 			} else if (arg.TypedValue.ArgumentType.IsArray) {
-				throw ErrorHelper.CreateError (1059, "Internal error: failed to instantiate mock attribute '{0}' (unknown type for the named argument #{1} ({2}). Please file a bug report (https://github.com/xamarin/xamarin-macios/issues/new) with a test case.", attribType.FullName, i + 1, arg.MemberName);
+				throw ErrorHelper.CreateError (1073, attribType.FullName, i + 1, arg.MemberName);
```

Comment:
Could you explain why the error # changing from `1059` to `1073` here? It looks like error # is retained ^ :)

Reply:
Code 1059 is used for a different error message in other places. The error message for 1059 is : `Found * * attributes on the member *. At most one was expected.`